handlers/generic_file_upload_ce_attachment_v1/README.md

```diff
@@ -0,0 +1,74 @@
+# Generic File Upload CE Attachment
+    This handler is used to copy an attachment from a submission to a third-party using a generic api with token validation in it's url.
+
+## Info Values
+    [Api Server]
+       Kinetic platform api server: http://server:port/kinetic
+    [Api Username]   
+       Kinetic platform user username
+    [Api Password]   
+       Kinetic platform user password 
+    [Space Slug]  
+      Slug of the space where the handler should be imported 
+    [Destination Api]
+      The base url for your third-party destination
+    [Destination Token Name]
+      The token name to be attached to the url
+    [Destination Token Value]
+      the token value to be attached to the url  
+
+## Parameters
+    [Error Handling]
+        How to handle error conditions in the handler: raise the error, or return error message
+    [Space Slug]
+        Slug of the Space where the handler should be imported
```

Comment:
Sorry, I missed this last time.

Slug of the space that contains the attached file to copy